src/protocol/commands/_vFile_setfs.rs

```diff
@@ -14,8 +16,11 @@ impl<'a> ParseCommand<'a> for vFileSetfs {
 
         match body {
             [b':', body @ ..] => {
-                let pid = decode_hex(body).ok()?;
-                Some(vFileSetfs{pid})
+                let fs = match decode_hex(body).ok()? {
+                    0 => FsKind::Stub,
+                    pid => FsKind::Pid(NonZeroUsize::new(pid).unwrap()),
+                };
```

Comment:
```suggestion
                let fs = match NonZeroUsize::new(decode_hex(body).ok()?) {
                    None => FsKind::Stub,
                    Some(pid) => FsKind::Pid(pid),
                };
```

I'm almost certain the two will compile down to the same code, but it's always better to remove an unnecessary unwrap when possible.